packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -53,8 +52,8 @@ export interface ClipboardCopyProps extends Omit<React.HTMLProps<HTMLDivElement>
   isBlock?: boolean;
   /** Adds Clipboard Copy variant styles. */
   variant?: typeof ClipboardCopyVariant | 'inline' | 'expansion' | 'inline-compact';
-  /** Copy button popover position. */
-  position?: PopoverPosition | 'auto' | 'top' | 'bottom' | 'left' | 'right';
```

Comment:
Hm, this is technically breaking. Maybe we can merge #6177 which appends to the types instead? Then we can rekick the core bump in #6160.

Reply:
ok